### PR TITLE
chore(google-translate): update lang name & flag

### PR DESCRIPTION
**Pull request by neighborhood999**
Signed-off-by: Jie Peng <im@jiepeng.me>

## Description

- Update language name & flag

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `metadata` folder

### CODE REVIEW

File: extensions/google-translate/package.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
           "value": "zh-CN"
         },
         {
-          "title": "🇨🇳 Chinese Traditional",
+          "title": "🇹🇼 Taiwan Mandarin",
```
Comment (FezVrasta) [inline extensions/google-translate/package.json:86]: Taiwanese maybe?

**Review by FezVrasta — APPROVED** [review]

File: extensions/google-translate/package.json (hunk 1 above)
Comment (neighborhood999) [reply extensions/google-translate/package.json:86]: Thank you for approving PR. Yes, you're right, let me update it. 🙏

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @neighborhood999 👋 

Thanks for the update 💪 Could you merge main into your branch, it seems like you're missing the `CHANGELOG.md` file 😄 

In that file you can add something like: 
```
## [Fix] - 2022-03-25
- Updated icon and name of zh-TW
```

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by neighborhood999** [comment]
> Hi @neighborhood999 👋
> 
> Thanks for the update 💪 Could you merge main into your branch, it seems like you're missing the `CHANGELOG.md` file 😄
> 
> In that file you can add something like:
> 
> ```
> ## [Fix] - 2022-03-25
> - Updated icon and name of zh-TW
> ```
> 
> Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

Thank you for your review, I have already updated the CHANGELOG.

**Comment by pernielsentikaer** [comment]
Awesome @neighborhood999 🔥 

Sorry for the much longer responsetime, I'll keep an eye on the pulls from this request 😄 

Do you have time to add [Categories to package.json?](https://developers.raycast.com/basics/prepare-an-extension-for-store#categories)

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by neighborhood999** [comment]
> Awesome @neighborhood999 🔥
> 
> Sorry for the much longer responsetime, I'll keep an eye on the pulls from this request 😄
> 
> Do you have time to add [Categories to package.json?](https://developers.raycast.com/basics/prepare-an-extension-for-store#categories)
> 
> Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

Done 😄

**Review by gebeto — APPROVED** [review]
Looks good for me

**Pull request merged**